### PR TITLE
README update for onchain architecture

### PR DESCRIPTION
**Pull request by anomit**
This PR updates the README to cover the major upgrades in components, architecture and data processing capabilities of pooler according to the latest update on a decentralized protocol state as available in the branch [`onchain_pooler`](https://github.com/PowerLoom/pooler/tree/onchain_pooler).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,56 @@
...
+        - Pair contracts
+        - Individual tokens participating in pair contract
+    - aggregated over time periods
+        - 24 hours
+        - 7 days
+- Transactions containing `Swap`, `Mint`, and `Burn` events
+
+
+
+In its last release (link to github commit or tag here), pooler was a *self contained* system that would provide access to the above. The present implementation differs from that in some major ways:
```
Comment (SwaroopH) [inline README.md:46]: https://github.com/PowerLoom/pooler/releases/tag/v0.1.3-alpha

File: README.md [hunk 2]
```diff
@@ -77,227 +205,179 @@ We use [pre-commit hooks](https://pre-commit.com/) to ensure our code quality is
...
+    }
+    catch (e){
+      console.error('7d top pairs', e);
+    }
+```
+
+## Find us
 
-You can read more about Audit Protocol and the Uniswap v2 PoC in the  [Powerloom Protocol Overview document](https://www.notion.so/powerloom/PowerLoom-Protocol-Overview-c3bf9dd9323541118d46a4d8684565d1#8ad76b8362b341bcaa9b3ae9fe203271)
+* [Discord](https://discord.gg/QCpVhshdtt)
```
Comment (SwaroopH) [inline README.md:379]: Let's change to: https://powerloom.io/discord

**Review by SwaroopH — APPROVED** [review]
Small suggestions aside, looks great!

P.S. I skipped reviewing the code change in `trade_volume.py`

**Review by xadahiya — APPROVED** [review]
Looks pretty good!

**Pull request merged**